testsuite/smoke-test/src/aptos_cli/validator.rs

```diff
@@ -76,6 +81,267 @@ async fn test_show_validator_set() {
     );
 }
 
+async fn check_vote_to_elected(swarm: &mut LocalSwarm) -> (Option<u64>, Option<u64>) {
```

Comment:
I don't quite follow what this function was doing (Probably because I don't have enough context on the stuff you are testing here). Maybe add a comment saying what this function is doing and suppose to check?